packages/@tinacms/cli/src/cmds/compile/defaultSchema.ts

```diff
@@ -12,66 +12,72 @@ limitations under the License.
 */
 
 export const defaultSchema = `
-import { defineSchema, defineConfig } from "tinacms";
+import { defineSchema, defineConfig } from 'tinacms'
+import { client } from "/.tina/__generated__/client"
 
+
+const branch =
+  process.env.NEXT_PUBLIC_TINA_BRANCH ||
+  process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF ||
+  process.env.HEAD ||
+  'main'
 const schema = defineSchema({
+  // See https://tina.io/docs/tina-cloud/connecting-site/ for more information about this config
+  config: {
+    token: '<Your Read Only Token>' // generated on app.tina.io,
```

Comment:
After fixing that I get:
```
Installing Tina packages. This might take a moment... ✅
✘ [ERROR] Could not resolve "/.tina/__generated__/client"

    .tina/schema.ts:2:23:
      2 │ import { client } from '/.tina/__generated__/client'
        ╵                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  BuildSchemaError: Error: Build failed with 1 error:
.tina/schema.ts:2:23: ERROR: Could not resolve "/.tina/__generated__/client"
```

Reply:
Thanks! I fixed these two errors.